channel-email/src/test/java/com/synopsys/integration/alert/channel/email/validator/EmailGlobalConfigurationValidatorTest.java

```diff
@@ -13,7 +13,7 @@
 import com.synopsys.integration.alert.common.rest.model.ValidationResponseModel;
 import com.synopsys.integration.alert.service.email.model.EmailGlobalConfigModel;
 
-public class EmailGlobalConfigurationValidatorTest {
+class EmailGlobalConfigurationValidatorTest {
```

Comment:
What happened here with the removal of public?

Reply:
This is to remove some of the Sonar code smells, I spoke with Paulo a few weeks back and he suggested when returning to these old tests we should just remove them to cut down on the number of remaining smells.

Reply:
As an FYI the rule description is: JUnit5 test classes and methods should have default package visibility

Reply:
Interesting, I did not know this. Will make the switch as well in the future